modules/library/main/src/test/java/org/geotools/styling/SLDTransformerTest.java

```diff
@@ -834,6 +830,50 @@ public void testTextSymbolizerTransformOutAndInAndOutAgain() throws Exception {
         String secondExport = st.transform(firstImport);
 
     }
+
+    @Test
+    public void testTextSymbolizerTransformOutAndInAndOutAndInAgainWithEmptyFontFamily() throws Exception {
```

Comment:
for a test case this specific quote the bug number or people will think you are nuts